src/Microsoft.Data.SqlClient/tests/ManualTests/AlwaysEncrypted/SqlNullValues.cs

```diff
@@ -0,0 +1,212 @@
+                            Assert.True(value2 is System.Data.SqlTypes.SqlInt32, "Unexpected type");
+                            Assert.True(10 == ((System.Data.SqlTypes.SqlInt32)value2).Value, "Unexpected Value");
+                            Assert.True(param.Value is System.Data.SqlTypes.SqlInt32, "Unexpected Return value");
+                        }
+                        else
+                        {
+                            // Expect varbinary
+                            Assert.True(value1 is System.Data.SqlTypes.SqlBinary, "Unexpected type");
+                            Assert.True(value2 is System.Data.SqlTypes.SqlBinary, "Unexpected type");
+                            Assert.True(param.Value is System.Data.SqlTypes.SqlBinary, "Unexpected Return value");
+                        }
+                        break;
+                }
+            }
+        }
+
+        private void DropFunctions(SqlConnection sqlConnection)
+        {
+            string[] funcNames = {UdfName, UdfNameNotNull};
+
+            foreach (string funcName in funcNames)
+            {
+                using (SqlCommand cmd = new SqlCommand(string.Format("IF EXISTS (SELECT * FROM sys.objects WHERE name = '{0}') \n DROP FUNCTION {0}", funcName), sqlConnection))
+                {
+                    cmd.ExecuteNonQuery();
+                }
+            }
+        }
+
+        public void Dispose()
+        {
+            foreach (string connStrAE in DataTestUtility.AEConnStringsSetup)
+            {
+                using (SqlConnection sqlConnection = new SqlConnection(connStrAE))
+                {
+                    sqlConnection.Open();
+                    Table.DeleteData(fixture.SqlNullValuesTable.Name, sqlConnection);
+                    DropFunctions(sqlConnection);
+                }
+            }
+        }
+    }
+
+    public class NullValueTestsData : IEnumerable<object[]>
+    {
+        public IEnumerator<object[]> GetEnumerator()
+        {
+            foreach (string connStrAE in DataTestUtility.AEConnStrings)
+            {
+                foreach (int connStrSetting in Enum.GetValues(typeof(ConnectionStringSetting)))
+                {
+                    foreach (int commandColumnEncryption in Enum.GetValues(typeof(SqlCommandColumnEncryptionSetting)))
+                    {
+                        yield return new object[] { connStrAE, connStrSetting, commandColumnEncryption, ReturnValueSetting.NotNull };
+                        yield return new object[] { connStrAE, connStrSetting, commandColumnEncryption, ReturnValueSetting.Null };
+                    }
+
+                }
+            }
+        }
+        IEnumerator IEnumerable.GetEnumerator() => GetEnumerator();
+    }
+
+    public enum ConnectionStringSetting
+    {
+        None,
+        Enabled,
+        Disabled
+    };
+
+    public enum ReturnValueSetting
+    {
+        NotNull,
+        Null
+    };
+
+}
```

Comment:
New line EOF

Reply:
I tried locally but it looks like it is still there. How can I remove this from editor?

Reply:
You just need to add a return at the end of line 203.

Reply:
Oh, OK. Done :)